pkgs/by-name/li/linien-gui/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, python3
+, linien-common
+, linien-client
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
doesn't start
```
Traceback (most recent call last):
  File "/nix/store/8kvb3hhw9hvpzmjkp4i873f9rv11kz50-linien-gui-0.8.0/bin/.linien-wrapped", line 6, in <module>
    from linien_gui.app import run_application
  File "/nix/store/8kvb3hhw9hvpzmjkp4i873f9rv11kz50-linien-gui-0.8.0/lib/python3.11/site-packages/linien_gui/app.py", line 23, in <module>
    from linien_client.connection import LinienClient
  File "/nix/store/4hwmg9xyvn9k7hr4ij66pva6pyc60425-linien-client-0.8.0/lib/python3.11/site-packages/linien_client/connection.py", line 27, in <module>
    from linien_common.config import DEFAULT_SERVER_PORT
  File "/nix/store/iyzbhv1ll28798y91f075zk41nniqdhr-linien-common-0.8.0/lib/python3.11/site-packages/linien_common/config.py", line 21, in <module>
    from appdirs import AppDirs
ModuleNotFoundError: No module named 'appdirs'
```

Reply:
added `appdirs` to propagatedBuildInputs. Though it stops again,
```
./result/bin/linien
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

fish: Job 1, './result/bin/linien' terminated by signal SIGABRT (Abort)
```

Reply:
'appdirs' is now added